x11-themes/gtk-engines-qtpixmap/gtk-engines-qtpixmap-0.28-r2.ebuild

```diff
@@ -10,7 +10,7 @@ inherit autotools eutils gnome2
 
 MY_P="QtPixmap-${PV}"
 
-DESCRIPTION="A modified version of the original GTK pixmap engine which follows the KDE color scheme"
+DESCRIPTION="A modified version of the original GTK pixmap engine with the KDE color scheme"
```

Comment:
You could also trim away the unnecessary fill word at the beginning. KDE should be replaced by Plasma.